andela/ah-jarvis-frontend
# Ft reset password 159961950

#### What does this PR do?
Allow user to reset password

#### Description of Task to be completed?
- send password reset link to email
- Change password if the email is valid

#### How should this be manually tested?
```
git clone git@github.com:andela/ah-jarvis-frontend.git  -b ft-reset-password-159961950

cd ah-jarvis-frontend

yarn 

yarn sass:build

yarn start
```
- navigate to http://localhost:3000/reset/password and enter a valid email
- a link will be sent to the email.
- click on the link and it will redirect to http://localhost:3000/verify/<token><email>
- enter new password

#### What are the relevant pivotal tracker stories?
[#159961950 ](https://www.pivotaltracker.com/story/show/159961950) Users should be able to be presented with a reset password link

#### Screenshots
<img width="521" alt="screen shot 2018-09-11 at 15 36 51" src="https://user-images.githubusercontent.com/18376530/45360632-ce175880-b5d8-11e8-9003-76ec38f67a53.png">
<img width="524" alt="screen shot 2018-09-11 at 15 37 15" src="https://user-images.githubusercontent.com/18376530/45360633-ceafef00-b5d8-11e8-9d38-1f6ef31a663a.png">
<img width="515" alt="screen shot 2018-09-11 at 15 37 40" src="https://user-images.githubusercontent.com/18376530/45360634-ceafef00-b5d8-11e8-90ac-fceb7651d698.png">
<img width="421" alt="screen shot 2018-09-11 at 15 38 30" src="https://user-images.githubusercontent.com/18376530/45360635-ceafef00-b5d8-11e8-8d92-d20f96b4607a.png">
